tofhir-engine/src/main/scala/io/tofhir/engine/cli/CommandFactory.scala

```diff
@@ -10,6 +10,8 @@ object CommandFactory {
       case "extract-redcap-schemas" => new ExtractRedCapSchemas()
       case "reload" => new Reload()
       case "run" | "execute" => new Run()
+      case "list" => new ListRunningMappings()
+      case "stop" => new Stop()
```

Comment:
In the help menu, we have the following documentation for stop command:
**stop - Stop the execution of the MappingJob (if any).**

I think that we should update it because as far as I understand, we can stop individual mappings.